views/js/controller/Tools/assessmentActivity.js

```diff
@@ -16,187 +16,115 @@
-                filter: false,
-                model: deliveryListModel,
-                paginationStrategyTop : 'none',
-                paginationStrategyBottom : 'none',
-                selectable : true,
-                sortorder : 'asc',
-                sortby : 'label'
-            }, deliveryListModel);
-            $deliveryList.datatable('refresh');
-
-
-            $('.js-pause').on('click', function() {
-                var config;
-
-                config = {
-                    renderTo : $container,
-                    actionName : msg,
-                    reason : true,
-                    allowedResources: [],
-                    reasonRequired: true,
-                    categoriesSelector: cascadingComboBox(categories['pause'])
-                };
-
-                bulkActionPopup(config).on('ok', function(reason){
-                    doPause(reason);
+        start : function () {
+            var $container = $('.container.assessment-activity');
+
+            getAssessmentActivity(function (res) {
+                if (!res.success) {
+                    return handleError(res.error);
+                }
+
+                $container.append(assessmentActivityTpl(res.data));
+
+                $('button.pause-active-test-sessions', $container)
+                .on('click', function () {
+                    pauseActiveTestSessions(function (pauseActiveTestSessionsRes) {
+                        if (!pauseActiveTestSessionsRes.success) {
+                            return handleError();
+                        }
+
+                        feedback().success(pauseActiveTestSessionsRes.message);
+                    });
                 });
-            });
 
-            $('.js-activity-chart-interval').on('change', function () {
-                var interval = $(this).val();
-                activityGraph.refresh({
-                    graphConfig : {
-                        data : {
-                            url: url.route('completedAssessmentsData', 'Tools', 'taoProctoring', {'interval' : interval})
-                        },
-                        axis : {
-                            x : {
-                                tick : {
-                                    format: interval === 'day' ? '%H:%M' : '%m-%d'
-                                },
-                                label : {
-                                    text:  interval === 'day' ? __('Hours') : __('Days')
+                getCompletedAssessmentsConfig(function (completedAssessmentConfigRes) {
+
+                    if (!completedAssessmentConfigRes.success) {
+                        return handleError(completedAssessmentConfigRes.error);
+                    }
+
+                    // if (completedAssessmentConfigRes.data.completed_assessments_auto_refresh) {
+                    if (true) {
+                        interval = setInterval(function () {
```

Comment:
please use the core/polling module for this.
It allow better manage the life cycle of the polling.